frontend/src/hooks/api/groups/types.ts

```diff
@@ -0,0 +1,36 @@
+import { TOrgRole } from "../roles/types";
+
+// TODO: rectify/standardize types
+
+export type TGroupOrgMembership = TGroup & {
+  customRole?: TOrgRole;
+}
```

Comment:
There's also formatting issues in this file

Reply:
Will resolve formatting issues with `prettier` in a separate PR.